html/browser.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>DTDI Mineral Observation Map</title>
 
-    <script src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.1/handlebars.min.js"></script>
+    <script src="handlebars.min.js"></script>
```

Comment:
For some reason the does not work locally on my machine, so I download the JS script and put it in the same folder to use it.